### PR TITLE
Create Apoorv38.md

### PR DESCRIPTION
**Pull request by Apoorv38**
I am creating a puppy who will accompany its owner wherever he/she will go, only difference is that it is human-made.

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

**Comment by bellesea** [comment]
heyy thanks so much for making a PR! we're back on tuesday and will start reviewing PRs in the order that they came on. 

to make the process smoother, please ensure you double checked that your file is properly formatted, you have all the parts you need, and that your plan is detailed enough for us to imagine how you're using the components in your budget to build your project and visualise you work on it :)

and fyi, we're looking for projects where/that:
- you can spend at least 10 days building (from Feb 14th to 23rd)
- you are able to modify or make your own in some ways
- you can learn something new from
- involves working with electronics or software on a lower level than a pre-assembled kit of parts

and, we normally don’t approve projects where you buy consumer components w/o modification like keyboards (buying PCB, keycaps, switches, and assembling it), PCs (buying motherboard, cpu, memory and plugging it together), and kits (eg. closed-sourced consumer drone kits). 

_(note: from a first glance of your project it looks like your budget might not be properly formatted? please let me know if you need any help with this :) also, can you please add to why you're interested in building this? thanks!)_

our Jan 15 11:59pm ET deadline is for PRs to come in and not for final approval so we'll work with you to either accept this project OR give you a chance to submit another PR. 

if you have any questions, please do not hesitate to ask her or DM me (i'm belle on slack), thank you!

File: Apoorv38.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+I'm not sure if it would be as good as I think but it's worth trying.
+
+## Plan
+
+In each leg I will use 3 servo-motors one for controlling the direction of   the pupp, the other two are for forward motion.
+I will use IR proximity sensor and Ultrasonic distance finder to avoid   obstalcesfrom a safe distance.
+I will use esp 32 cam for face recognition of the owner.
+I'll use OLED screen for its face.
+It would be powered using a Lithium Polymer battery(800 mAh).
+I am using two Arduino boards because I think 1 wouldn't be enough for the pupp.
```
Comment (exu3) [inline Apoorv38.md:22]: How are the 2 Arduino boards going to communicate with each other? Are they networked?

File: Apoorv38.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+|LiPo balance charger|https://robu.in/product/b3ac-compact-balance-charger-2s-3s-lipo/| $4.50 |
+|3v LDO voltage regulator x4|https://robu.in/product/lds3985m33r-3-3v-300ma-ldo-bicmos-voltage-regulator-5-pin-sot-23-stmicroelectronics/| $4 |
+|Multimeter|https://robu.in/product/mastech-mas830l-digital-multimeter-with-probes-original/| $11|
+|Poteniometer x4|https://robu.in/product/3310y-001-503l-rotary-potentiometer/| $12.50 |
+|Ultrasonic range finder HC-SR04|https://robu.in/product/hc-sr04-ultrasonic-range-finder/| $1 |
+|Logic level convertor|https://robu.in/product/8-channel-bi-directional-logic-level-converter-module/| $1 |
+|Jumper wires male to male x2|https://robu.in/product/male-to-male-jumper-wires-40pcs-20cm/| $1 |
+|Pin male header strip|https://robu.in/product/2-54mm-1x40-pin-male-single-row-straight-short-header-strip-pack-of-3/?pid=1114172| $0.50 |
+|Glue Gun|https://amzn.eu/d/6Jnao54| $5 |
+|Drill|https://amzn.eu/d/7eHzw34| $18 |
```
Comment (exu3) [inline Apoorv38.md:48]: What's the drill for?

**Review by exu3 — CHANGES_REQUESTED** [review]
Hello! Thanks for submitting this proposal. Sounds like a neat project. Some questions that come to mind:
- Are you designing your own 3D models? If not, where will you get them from?
- based on your proposal, I don't have a clear idea of what this might look like. How big will it be? What will it look like?
- Are you following a tutorial or guide for this? If so, could you link it here?

**Comment by exu3** [comment]
Hi @Apoorv38, were you still interested in pursuing this project? I've left some comments above, let me know if you have any questions.

**Comment by Apoorv38** [comment]
Yes, I am still interested in the project.
I will do it by today.(my clock)

File: Apoorv38.md (hunk 1 above)
Comment (Apoorv38) [reply Apoorv38.md:22]: Using software serial.

File: Apoorv38.md (hunk 2 above)
Comment (Apoorv38) [reply Apoorv38.md:48]: I need it to make holes in my 3D printed parts.(It is not that necessary but its better to have one)

**Review by exu3 — APPROVED** [review]
Looks good to me. The @hackclub/bank-ops team will review and follow up with next steps via Slack.

Happy hacking!

**Review by daisybanks — APPROVED** [review]
Hi @Apoorv38! This sounds so awesome and fun! What will you name this buddy once it's built??? Can you please comment your Slack handle or dm on Slack @daisy so we can talk about next steps??

**Comment by Apoorv38** [comment]
Sure!

**Pull request merged**